src/main/java/net/sf/jabref/logic/CustomEntryTypesManager.java

```diff
@@ -1,30 +1,28 @@
 package net.sf.jabref.logic;
 
-import java.util.ArrayList;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Optional;
+import java.util.stream.Collectors;
 
 import net.sf.jabref.model.EntryTypes;
 import net.sf.jabref.model.database.BibDatabaseMode;
 import net.sf.jabref.model.entry.CustomEntryType;
-import net.sf.jabref.model.entry.EntryType;
 import net.sf.jabref.preferences.JabRefPreferences;
 
 public class CustomEntryTypesManager {
 
-    public static final List<EntryType> ALL = new ArrayList<>();
     /**
      * Load all custom entry types from preferences. This method is
      * called from JabRef when the program starts.
      */
     public static void loadCustomEntryTypes(JabRefPreferences prefs) {
```

Comment:
Could you change this class to be independent of `JabRefPreferences`? This would get us one step closer to our desired architecture. For the load method, this is easy (just put in CustomEntryType instead of the preferences). For the saving it is a little more tricky and you will need to push code into the calling UI-class.